vendor/k8s.io/kubernetes/openshift-kube-apiserver/openshiftkubeapiserver/patch.go

```diff
@@ -128,6 +128,10 @@ func NewOpenShiftKubeAPIServerConfigPatch(delegateAPIServer genericapiserver.Del
 		// END CONSTRUCT DELEGATE
 
 		patchContext.informerStartFuncs = append(patchContext.informerStartFuncs, kubeAPIServerInformers.Start)
+		patchContext.postStartHooks["openshift.io-kubernetes-informers-synched"] = func(context genericapiserver.PostStartHookContext) error {
+			kubeInformers.WaitForCacheSync(context.StopCh)
```

Comment:
Should this fail if you cannot sync?

Reply:
potentially yes. But we seem to be very undecided elsewhere.

Reply:
If I see it correctly, this can only return bools in the map if stopCh closed. In that case, we don't care about the result.